src/metrics/halstead.rs

```diff
@@ -76,39 +113,37 @@ impl<'a> fmt::Display for Stats<'a> {
     }
 }
 
-impl<'a> Stats<'a> {
+impl Stats {
     /// Merges a second `Halstead` metric suite into the first one
-    pub fn merge(&mut self, other: &Stats<'a>) {
-        for (k, v) in other.operators.iter() {
-            *self.operators.entry(*k).or_insert(0) += v;
-        }
-        for (k, v) in other.operands.iter() {
-            *self.operands.entry(*k).or_insert(0) += v;
-        }
+    pub fn merge(&mut self, other: &Stats) {
```

Comment:
we shouldn't have this function.
You can't sum the u_***: the maps need to be merged and then the new values are the length of the new maps.